AutomatticTracks/src/main/java/com/automattic/android/tracks/DeviceInformation.java

```diff
@@ -125,6 +125,7 @@ public DeviceInformation(Context context) {
             mImmutableDeviceInfoJSON.put("app_name", getAppName());
             mImmutableDeviceInfoJSON.put("app_version", getAppVersionName());
             mImmutableDeviceInfoJSON.put("app_version_code", Integer.toString(getAppVersionCode()));
+            mImmutableDeviceInfoJSON.put("language", getDeviceLanguage());
```

Comment:
All the fields returned from `DeviceInformation` are prefixed with `device_info` when sent to the server. Resulting in something like the following structure:
```
           [device_info_os] => Android
           [device_info_os_version] => 5.0
           [device_info_manufacturer] => Genymotion
           [device_info_brand] => generic
           [device_info_model] => Google Nexus 6 - 5.0.0 - API 21 - 1440x2560
           [device_info_app_name] => WordPress
           [device_info_app_version] => 4.1-rc-3
           [device_info_app_version_code] => 184
           [device_info_has_nfc] => false
           [device_info_has_telephony] => true
           [device_info_display_density_dpi] => 560
           [device_info_bluetooth_version] => none
           [device_info_current_network_operator] => 
           [device_info_phone_radio_type] => gsm
           [device_info_wifi_connected] => true
           [device_info_language] => en_US
```